docs/pages/validator-management/external-signer.md

```diff
@@ -0,0 +1,23 @@
+# External Signer
+
+Lodestar supports connecting an external signing server like [Web3Signer](https://docs.web3signer.consensys.io/), [Diva](https://docs.shamirlabs.org/),
+or any other service implementing the [remote signing specification](https://github.com/ethereum/remote-signing-api). This allows the validator client
+to operate without storing any validator private keys locally by delegating the signing of messages (e.g. attestations, beacon blocks) to the external signer
+which is accessed through a [REST API](https://ethereum.github.io/remote-signing-api/) via HTTP(S). This API should not be exposed directly to the public
+Internet and appropriate firewall rules should be in place to restrict access only from the validator client.
+
+## Configuration
+
+Lodestar provides [CLI options](./validator-cli.md#--externalsignerurl) to connect an external signer.
```

Comment:
```suggestion
Lodestar provides [CLI options](./validator-cli.md#--externalsigner) to connect an external signer.
```

I think this anchor tag comes from the group name that you changed in this PR

Reply:
We are not using the group in docs, we could add another heading for it in [markdown.ts](https://github.com/ChainSafe/lodestar/blob/unstable/packages/cli/docsgen/markdown.ts) but not sure if it makes too much sense, maybe if we add a top-level description for the group with a short description about the group of flags.

The group tag is only relevant for the CLI when printing out options via `--help`

![image](https://github.com/ChainSafe/lodestar/assets/38436224/a3321f35-4588-462d-b6d1-d9e60af5594b)